quic/s2n-quic-transport/src/ack.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod interest {
```

Comment:
Can you make this a separate file? Having the entire thing nested isn't great. Also could you reexport core::ack here? Makes it easy to only have to import one `ack` module.